src/In.ProjectEKA.HipService/Link/Patient/LinkPatient.cs

```diff
@@ -29,31 +29,24 @@ public class LinkPatient : ILink
             PatientLinkReferenceRequest request)
         {
             var (patient, error) = PatientAndCareContextValidation(request);
-            if (error != null)
-            {
-                return new Tuple<PatientLinkReferenceResponse, ErrorResponse>(null, error);
-            }
+            if (error != null) return new Tuple<PatientLinkReferenceResponse, ErrorResponse>(null, error);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4839081)